libraries/AP_HAL_ChibiOS/hwdef/FlyingMoonF407/hwdef.dat

```diff
@@ -175,3 +175,5 @@ PE5 TIM9_CH1 TIM9 ALARM
 #  IRQ for MPU6000
 PB0 DRDY_ICM20689 INPUT PULLUP
 PB1 DRDY_IST8310 INPUT PULLUP
+
+include ../include/minimize_common.inc
```

Comment:
This .inc shouldn't be included except by `minimize_features.inc` or `minimize_fpv_osd.inc` - or a new inc if you think it's worthwhile.  Should probably be more strenuous in the comments at the top of `minimize_common.inc`?

Reply:
changed to minimize_features